src/corpus/ui/Chapters.tsx

```diff
@@ -169,7 +176,17 @@ const Manuscripts = withData<
                   <ul className="list-of-manuscripts__uncertain-fragments">
                     {uncertainFragments.map((fragment, index) => (
                       <li key={index}>
-                        <FragmentariumLink item={fragment} />
+                        <FragmentariumLink
+                          item={{
```

Comment:
Would it be more readable if we extract it to `const item = ...` and then call `<FragmentariumLink item={item} />`?

Reply:
`uncertainFragment` instead `fragment`